4337/README.md

```diff
@@ -156,9 +156,9 @@ npm run test:e2e
 
 ### Deploy
 
-> :warning: **Make sure to use the correct commit when deploying the contracts.** Any change (even comments) within the contract files will result in different addresses. The tagged versions used by the Safe team can be found in the [releases](https://github.com/safe-modules/releases).
+> :warning: **Make sure to use the correct commit when deploying the contracts.** Any change (even comments) within the contract files will result in different addresses. The tagged versions used by the Safe team can be found in the [releases](https://github.com/safe-global/safe-modules/releases).
```

Comment:
Currently our releases are not up to date, something we need to follow up on

Reply:
If we plan to have more modules in the monorepo, I would employ the same setup with changelogs and stuff as we did for the safe-apps-sdk using changesets: https://github.com/safe-global/safe-apps-sdk/releases